plugins/modules/source_control/gitlab/gitlab_group_members.py

```diff
@@ -0,0 +1,230 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2020, Zainab Alsaffar <Zainab.Alsaffar@mail.rit.edu>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: gitlab_group_members
+short_description: Manage group members on GitLab Server
+description:
+    - This module allows to add and remove members to/from a group, or change a member's access level in a group on GitLab.
+version_added: '1.2.0'
+author: Zainab Alsaffar (@zanssa)
+requirements:
+    - python-gitlab python module <= 1.15.0
+    - administrator rights on the GitLab server
+options:
+    server_url:
+        description:
+            - URL of a GitLab server.
+        required: true
+        type: str
+    access_token:
```

Comment:
All other GitLab modules call this `api_token`:
```suggestion
    api_token:
```

Reply:
Changed it to api_token & modified the integration test playbook to reflect that!